src/App.jsx

```diff
@@ -1,35 +1,20 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
-
+  const todos = [
```

Comment:
Remember to always format you code. Format as you code and format before submitting a PR. You will start noticing the difficulty in reading unformatted code as your codebase grows. IDEs usually have a handy hotkey combo or you can find it in the dropdown menu. (Or maybe a third part library has been installed)